src/components/views/rooms/EventTile.tsx

```diff
@@ -1362,22 +1362,9 @@ export default class EventTile extends React.Component<IProps, IState> {
                         >
                             { linkedTimestamp }
                             { this.renderE2EPadlock() }
-                            { replyChain }
-                            <EventTileType ref={this.tile}
-                                mxEvent={this.props.mxEvent}
-                                forExport={this.props.forExport}
-                                replacingEventId={this.props.replacingEventId}
-                                editState={this.props.editState}
-                                highlights={this.props.highlights}
-                                highlightLink={this.props.highlightLink}
-                                showUrlPreview={this.props.showUrlPreview}
-                                permalinkCreator={this.props.permalinkCreator}
-                                onHeightChanged={this.props.onHeightChanged}
-                                callEventGrouper={this.props.callEventGrouper}
-                                tileShape={this.props.tileShape}
-                                getRelationsForEvent={this.props.getRelationsForEvent}
-                            />
-                            { keyRequestInfo }
+                            <div className="mx_EventTile_body">
+                                { MessagePreviewStore.instance.generatePreviewForEvent(this.props.mxEvent) }
```

Comment:
I am a little confused by this change... Why are we doing this? It seems like the `callEventGrouper` isn't getting to the event?

Reply:
There won't be any call events in a thread. This is to tackle this https://github.com/vector-im/element-web/issues/19710#issuecomment-975730760

Reply:
Oh, oops, didn't notice the full context of the code :facepalm: